specification/cognitiveservices/AnomalyDetector/multivariate/routes.tsp

```diff
@@ -35,38 +49,44 @@ Storage folder that contains multiple CSV files, where each CSV file has
 two columns, time stamp and variable. Or the Blob Storage URI can point to a single blob that contains a CSV file that has all the variables and a
 time stamp column.
 """)
-// To improve CADL implementation apply: https://github.com/Azure/azure-rest-api-specs/issues/21527
-op TrainMultivariateModel(
-  @doc("Model information.")
-  @body modelInfo: ModelInfo,
-): {
-  @statusCode statusCode: 201,
-  @doc("Location and ID of the model.")
-  @header Location: string,
-  @body result: AnomalyDetectionModel
-  } | ResponseError;
+op TrainMultivariateModel is MultivariateServiceAction<
+  // TParams
+  {
+    @doc("Model information.")
+    @body modelInfo: ModelInfo,
```

Comment:
We should avoid the use of `@body`. Is there a specific reason you need it here?

Reply:
It turns out that we do need it because if I just spread the model then the body is not defined as required.

Reply:
I don't understand what  you mean. The only difference is the name of the body parameter, no? Are you seeing some other difference?

Reply:
There are other differences. Here's the diff:
```diff
         "parameters": [
           {
-            "name": "modelInfo",
+            "name": "body",
             "in": "body",
-            "required": true,
-            "description": "Model information.",
             "schema": {
               "$ref": "#/definitions/Multivariate.ModelInfo"
             }
```

Reply:
The required missing is an autorest-swagger emitter bug. Please open one. The description should either come from the model or the operation. The boiler-plate style that only repeats what is already in the model information is not very helpful to our customers.

In my opinion, neither of these issues would lead me down the path of using `@body` in the specification since that is ultimately going to interact in interesting ways with the metadata APIs.